fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -96925,8 +97222,8 @@ def: "Specialised cell of the adult midgut found in the anterior and middle midg
 subset: cur
 synonym: "adult cuprophilic cell" EXACT [FlyBase:FBrf0139611]
 is_a: FBbt:00005782 ! copper cell
-relationship: part_of FBbt:00110898 ! adult midgut anterior region R3a
-relationship: part_of FBbt:00110899 ! adult midgut anterior region R3b
+relationship: part_of FBbt:00110898 ! adult midgut region R3a
+relationship: part_of FBbt:00110899 ! adult midgut region R3b
```

Comment:
Not sure this works.  The OWL says "**all** adult copper cells are part_of R3a _**and**_ R3b" but presumably some are part of 3a and some of 3b.  Is there a general term for R3 you can use instead?